lib/hanami/utils/string.rb

```diff
@@ -141,6 +147,7 @@ def capitalize
       #   string = Hanami::Utils::String.new 'hanami_utils'
       #   string.classify # => 'HanamiUtils'
       def classify
+        return self if classified?
```

Comment:
@jodosha I will take a look at the regex/method again if I can make it without it.